components/health_controller/src/reporters.rs

```diff
@@ -61,13 +62,26 @@ impl UnifiedSlowScore {
     pub fn new(cfg: &RaftstoreReporterConfig) -> Self {
         let mut unified_slow_score = UnifiedSlowScore::default();
         // The first factor is for Raft Disk I/O.
-        unified_slow_score
-            .factors
-            .push(SlowScore::new(cfg.inspect_interval));
+        unified_slow_score.factors.push(SlowScore::new(
+            cfg.inspect_interval,
+            cfg.inspect_interval,
```

Comment:
pass two?

Reply:
yes

Reply:
Recommends to add some necessary annotations here to make it more clearly to read.
```suggestion
            cfg.inspect_interval, /* timeout */
            cfg.inspect_interval, /* inspect interval */
```